### PR TITLE
Update turtlecoin-pools.json

### PR DESCRIPTION
**Pull request by crazy7bird**
Hello, i d like to add my pool.
A little pool with TURTL mining game project.

### CODE REVIEW

**Comment by RocksteadyTC** [comment]
what is your username in Discord?

**Comment by crazy7bird** [comment]
crazybird

**Comment by derp-derp-derp** [comment]
@RocksteadyTC I'll leave this PR merge to you, or at least to your approval.

**Comment by crazy7bird** [comment]
Sorry i dont understand what should i do ? (I m not a big user off github )

**Comment by DiscoTim** [comment]
you could do `http://www.crazybird.fr:8117/` for your api if you wanted

**Comment by derp-derp-derp** [comment]
Thanks, Tim. I've updated it.

File: v2/turtlecoin-pools.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         {
             "name" : "Crazybird's Pool",
             "url" : "http://www.crazybird.fr/",
-            "api" : "http://vps573884.ovh.net:8117/",
+            "api" : "http://www.crazybird.fr:8117/",
```
Comment (crazy7bird) [inline v2/turtlecoin-pools.json]: You'r right, that more understandable. Same if both adress point to the same  server.
Thanks

**Comment by derp-derp-derp** [comment]
@RocksteadyTC is it cool if I merge this single-user pool?

**Comment by derp-derp-derp** [comment]
@RocksteadyTC There is only one user connected to it - the owner presumably.

**Comment by crazy7bird** [comment]
@RocksteadyTC , @derp-derp-derp 
Yes for the moment it's a single user pool (i'm not a good at hiring miners).
But may attract your interest ? 
With this pool i'm actualy devlopping a minigame based on mining hashrate.
I made this minigame with coinhive, then i switch to TURTL, the pool help me to get the mining stat of the player, and uses them as an input for the game.
For the moment, the game dev is realy slow due to my main job and my newborn, but im still on it !!.

Ho for test the game, you have to mine for me (i could set a temporary TRTL adress on my mining hardware, that i communicate to you for testing the game if you wish), then enter your adress for get stats on main page an then go to crazybird.fr/dev/ and clik on minigame.

**Pull request merged**